wiremock-docker/mappings/basm_api_location_MULTIPLE_ON_DATE.json

```diff
@@ -1,6 +1,11 @@
 {
   "request" : {
-    "url" : "/api/reference/locations?filter%5Bcreated_at%5D=2021-05-05&per_page=100",
+    "urlPathPattern": "/api/reference/locations",
+    "queryParameters" : {
+      "filter%5Bcreated_at%5D" : {
+        "matches" : "^\\d{4}-\\d{2}-\\d{2}$"
```

Comment:
This is a crude regex but fine for local testing.  The point at which it is actually called the application will only ever provide a valid date.  It doesn't really care what the date is.  The response will always default to the current date.  So WireMock can pull through query param values but sadly not in the format used by BaSM.